### PR TITLE
Copied in nodeschool cairo and minor edits

### PR DESCRIPTION
**Pull request by bmann**
So, this is wrong, because a gh-pages branch needs to be created on nodeschool/vancouver first. If someone can do that, I can resubmit this, or something. I don't have a lot of experience with PRs.

But, by creating gh-pages and putting something like the contents of what I have in here, we will have a web presence up at nodeschool.io/vancouver/

I copied the NodeSchool Cairo instance -- viewable at http://nodeschool.io/cairo/

The index and sub pages and such need to be beefed up to be layouts and such so we can easily use Jekyll to create event pages and other info. I can help with this.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <link rel="icon" href="favicon.ico">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="olizilla"/>
+  <link rel="stylesheet" href="dark-style.css"/>
+  <title>Nodeschool Cairo</title>
```
Comment (gyaresu) [inline index.html]: Reference to Cairo.

**Comment by gyaresu** [comment]
Great idea @bmann 
I can create the branch and am also quite familiar with jekyll so willing to help out.

If you're happy with spending time on this then that would be great!
There's a few things in the meta names etc. that could be removed/replaced but as you say this can be a start and we can change as we go.

Nice one.

**Comment by sintaxi** [comment]
cool. good call @bmann

**Comment by gyaresu** [comment]
And I just read up about changing the base branch of your pull request. 

Turns out it's not possible, so if you're happy to setup the page and resubmit then that would be golden.

And until the pull request you submit is merged you can continue to push to the submission and whatever your most recent version is will be the one that gets merged.

**Comment by lvivier** [comment]
@bmann :+1: right on, looks good
@gyaresu we can also just merge with git :)

**Comment by bmann** [comment]
I'm closing this pull request in favour of #5 which correctly pulls against gh-pages branch.

**Pull request closed without merging**